src/query.ts

```diff
@@ -56,6 +65,13 @@ async function run(): Promise<void> {
       chdir(workDir);
 
       await uploadError(error, repo, artifactClient);
```

Comment:
My understanding was that we still wanted to keep all the old behaviour (in parallel with the extra live results steps), and then only separate the flows once live results is fully "live"? 

I may have misunderstood though, so happy to separate them out here instead, if that makes more sense! 😅

Reply:
Hmm yeah you're right - I think that's what some of us had in mind. But I'm wondering if it still makes sense 🤔 why would I want the artifacts for example if I'm using the new flow?

Reply:
> Hmm yeah you're right - I think that's what some of us had in mind. But I'm wondering if it still makes sense 🤔 why would I want the artifacts for example if I'm using the new flow?

I guess it would give us a smoother way to transition between the old/new flow in VS Code too? It might be useful to compare the results in both cases 🤔

Reply:
> compare the results in both cases

As in the actual SARIF files?

Reply:
> > compare the results in both cases
> 
> As in the actual SARIF files?

Basically, yes. We could make sure that the results displayed in the old results view match the results in the new view (i.e. that nothing got lost along the way). I appreciate that this might be overkill though 😅

Reply:
It feels like it to me because we're not changing anything at that level really. 

But happy either way. @robertbrignull keen to hear your thoughts.

Reply:
I don't have much opinion on whether we do both uploads for now or if it's an if-else. I'd be tempted to stick with doing both right now so this PR doesn't get more tangled. Then perhaps closer to the release when we're more confident we switch it to being if-else. It would be nice to do that before the public beta release itself so then the behaviour is fully controller by the dotcom feature flag and we don't have to worry about making code changes here in sync with that.